src/rhsm/certificate.py

```diff
@@ -562,7 +562,7 @@ def write(self, pem_path):
         f.write(self.content)
         self.path = pem_path
         f.close()
-        os.chmod(pem_path, 0600)
+        os.chmod(pem_path, 0o600)
```

Comment:
Yes. This change was done by 2to3. See https://www.python.org/dev/peps/pep-3127/#removal-of-old-octal-syntax

We *could* use the constants, but personally, I find them far less readable than the octal.